helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -300,6 +300,9 @@ public void addJob(final Job job) throws JobExistsException {
         }
         throw new JobExistsException(id.toString());
       }
+    } catch (NoNodeException e) {
```

Comment:
Should we add an `ensurePath()` for those paths as well? Does this happen when the master is slow to startup?